src/services/store.ts

```diff
@@ -2,11 +2,12 @@ import pluginJson from '../plugin.json';
 
 const SERVICES_LOCALSTORAGE_KEY = `${pluginJson.id}.services.favorite`;
 
-export function getFavoriteServicesFromStorage(ds: any): string[] {
-  if (!ds || typeof ds !== 'string') {
+// This should be a string, but we'll accept anything and return an empty array if it's not a string
```

Comment:
Why? Shouldn't it be string or null?

Reply:
It can be `VariableValue | null | undefined` and `VariableValue` can be array of basically anything.

Reply:
Oh, is coming from there :(